sysl2/sysl/Parser_test.go

```diff
@@ -13,39 +14,46 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func pyParse(filename, root string) (*sysl.Module, error) {
-	output := filename + ".pb"
+func readSyslModule(filename string) (*sysl.Module, error) {
+	var buf bytes.Buffer
 
-	args := []string{"pb", "-o", output, filename}
+	f, err := os.Open(filename)
+	if err != nil {
+		return nil, err
+	}
+	io.Copy(&buf, f)
+	f.Close()
+
+	module := &sysl.Module{}
+	if err := proto.UnmarshalText(buf.String(), module); err != nil {
+		return nil, err
+	}
+	return module, nil
+}
+
+func pySysl() string {
+	if pySysl, found := os.LookupEnv("SYSL_PYTHON_BIN"); found {
```

Comment:
Nit: I believe go convention is to call the var above `ok` not `found` (see [docs](https://golang.org/pkg/os/#LookupEnv))

Reply:
Done. I also reworked it into a computed-once variable instead of a function call.